### DISCUSSION TITLE
No DTU connection when using ethernet

### DISCUSSION THREAD

**Discussion by Ascendor** [tbnobody/OpenDTU#449] [Q&A]
Can anyone make sense of this?:

I have a perfectly working setup. Getting data consistently every 5sec, so seems stable. As soon as I plug in ethernet and use this connection: No DTU data anymore. (Meaning: Not a single update at all!)
As soon as I pull the plug: Data connection returns, as stable as before

(NRF+ Modul is connected, NTP is synced)

Any ideas?

**Comment by raccoon0815** [comment]
a capacitor might help with the standard NRF24L01+ modules

if you use the olimenx ESP32-POE with ISO  (ESP32-POE -ISO) board...  here I would use the ebyte E01-ML01DP5 module

check for more information https://github.com/tbnobody/OpenDTU/discussions/339
